scripts/script.yaml

```diff
@@ -502,23 +503,45 @@
 
     - name: current-report-temperature
       steps:
-        - say: האם נמדד חום ביממה האחרונה?
-        - wait:
-            options:
-            - show: "כן, נמדד חום"
-              value: "yes"
+        - switch:
+            arg: general_feeling
+            cases:
+            - default: true 
```

Comment:
why do we need a default?

Reply:
...not sure, but that what made my script run, while trying to debug it. Will recheck and omit if unecessery